src/View/Helper/FormElement.php

```diff
@@ -20,7 +20,7 @@ class FormElement extends BaseAbstractHelper
     /**
      * Instance map to view helper
      *
-     * @var array
+     * @var array<class-string, string>
```

Comment:
```suggestion
     * @var array<class-string<\Laminas\Form\ElementInterface>, string>
```
And same for `addClass()` param